modules/ROOT/pages/authentication-authorization/limitations.adoc

```diff
@@ -392,7 +392,8 @@ So due to the additional data access required by the security checks, this opera
 === Property-based access control limitations
 Extra node or relationship-level security checks are necessary when adding security rules based on property rules, and these can have a significant performance impact.
 
-The following example shows how the database behaves when adding security rules for nodes to roles `restricted` and `unrestricted`, the same limitations apply to relationships as well:
+The following example shows how the database behaves when adding security rules for nodes to roles `restricted` and `unrestricted`. 
+The same limitations apply to relationships:
```

Comment:
```suggestion
The same limitations apply to relationships.
```
feels odd to have `:` on this line now since the example below ore belongs with the sentence before it

Reply:
I agree